examples/full.py

```diff
@@ -3,178 +3,230 @@
 import re
 
 from flask import Flask
-from slackify import (ACK, OK, Slackify, async_task, block_reply,
-                      request, respond, text_block, Slack)
-
+from slackify import (
+	ACK, OK, Slackify, async_task, block_reply,
+	request, respond, text_block, reply_text, Slack)
 
 app = Flask(__name__)
-slackify = Slackify(app=app)
+slackify = Slackify(app = app)
```

Comment:
Python’s [PEP 8 recommendation](https://www.python.org/dev/peps/pep-0008/#other-recommendations) is without spaces in between:

> Don’t use spaces around the = sign when used to indicate a keyword argument, or when used to indicate a default value for an unannotated function parameter:

Reply:
I believe I got this fixed on the last push. Thanks!